source/manual/mapit-database-not-available.html.md

```diff
@@ -0,0 +1,83 @@
+---
+owner_slack: "#re-govuk"
+title: Fix Mapit database not available
+layout: manual_layout
+parent: "/manual.html"
+section: Databases
+last_reviewed_on: 2020-11-02
+review_in: 6 months
+---
+
+Each Mapit machine has its own PostgreSQL database which powers the Mapit
+application. When the machine is first created by AWS, the latest data is
+[supposed to be imported into the database][autoimport].
+
+[autoimport]: https://github.com/alphagov/govuk-puppet/pull/9657
+
+This feature is currently not working which means that new Mapit machines are
+not added to the load balancer until the data has been imported manually. There
+will also be numerous errors in Icinga related to 5xx errors and events in
+Sentry:
+
+```
+OperationalError: could not connect to server: Connection refused
+        Is the server running on host "127.0.0.1" and accepting
+        TCP/IP connections on port 5432?
+```
+
+> RE are aware that the automatic import process doesn't work.
```

Comment:
Could you add something about whether they intend to fix it, and to what sort of timescale? Is there a Trello card we can link to?

Reply:
@rtrinque Do you have any thoughts on this above? ^

It came from the work we did to scale up the Mapit machines a couple of weeks ago.